backend/inspirehep/assign/views.py

```diff
@@ -82,3 +87,43 @@ def assign_view():
     if to_author_recid is None:
         return jsonify({"stub_author_id": stub_author_id}), 200
     return jsonify({"message": "Success"}), 200
+
+
+@blueprint.route("conference", methods=["POST"])
+@login_required_with_roles([Roles.cataloger.value])
+@parser.use_args(
+    {
+        "conference_recid": fields.String(required=True),
+        "literature_recids": fields.List(fields.String, required=True),
+    },
+    locations=("json",),
+)
+def literature_assign_conferences_view(args):
+    conference_recid = args["conference_recid"]
+    literature_recids = args["literature_recids"]
+
+    if not conference_recid or not literature_recids:
+        return (
+            jsonify(
+                {
+                    "message": "conference_recid and literature_recids are required parameters."
+                }
+            ),
+            400,
+        )
+
+    if not isinstance(literature_recids, list):
+        return (
+            jsonify(
+                {"message": "literature_recids has to be a list of literature recids"}
+            ),
+            400,
+        )
```

Comment:
we don't need these checks with the parser

Reply:
Ah, yes. You are right ;)